classes/Customer.php

```diff
@@ -180,8 +180,8 @@ class CustomerCore extends ObjectModel
         'primary' => 'id_customer',
         'fields' => [
             'secure_key' => ['type' => self::TYPE_STRING, 'validate' => 'isMd5', 'copy_post' => false, 'size' => 32],
-            'lastname' => ['type' => self::TYPE_STRING, 'validate' => 'isCustomerName', 'required' => true, 'size' => 255],
-            'firstname' => ['type' => self::TYPE_STRING, 'validate' => 'isCustomerName', 'required' => true, 'size' => 255],
+            'lastname' => ['type' => self::TYPE_STRING, 'validate' => 'isCustomerName', 'size' => 255],
```

Comment:
This was blocked many times in https://github.com/PrestaShop/PrestaShop/pull/38988, why are you opening yet another PR with this change?

Reply:
Hello @Hlavtox, @PSebou discussed this topic with @kpodemski and @jolelievre and yes indeed this change is impactful. We thought it could be nice at the beginning however following feedbacks, the plan is to change this to allow empty values instead of "not required". Easier for database structure.